pkg/asset/installconfig/gcp/client.go

```diff
@@ -6,6 +6,7 @@ import (
 	"strings"
 	"time"
 
+	gcpv "github.com/openshift/installer/pkg/types/gcp/validation"
```

Comment:
We follow an order for imports so its best if we move this import to a new section below.

golang libraries

external libraries (non-openshift)

openshift libraries